src/lib/plot/axis/axis-ticks.js

```diff
@@ -51,16 +51,10 @@ function _getTickFormatFn(scale, tickTotal, tickFormat) {
     tickFormat;
 }
 
-function _getTickValues(scale, tickTotal, tickValues) {
-  return !tickValues ?
-    (scale.ticks ? scale.ticks(tickTotal) : scale.domain()) :
-    tickValues;
-}
-
 function AxisTicks(props) {
   const {
     attr,
-    orientation,
+    position,
```

Comment:
The d3 API refers to this as `orientation`. Any strong reason to change this to `position`?

Ref:
https://github.com/d3/d3-axis/blob/master/README.md#api-reference